volmdlr/step.py

```diff
@@ -183,23 +188,40 @@ def advanced_brep_shape_representation(arguments, object_dict):
     return shells
 
 
+def frame_map_closed_shell(closed_shells, item_defined_frames, shape_representation_frames):
+    for shell3d in closed_shells:
+        for f3d in item_defined_frames:  # item_defined_transformation
+            for frame3d in shape_representation_frames:  # shape_representation
+                if f3d.origin != volmdlr.O3D != frame3d.origin and frame3d != f3d:
+                    break
+            else:
+                continue
+            break
+        else:
+            continue
+        new_faces = [face.frame_mapping(frame3d, 'new') for face in shell3d.faces]
+        shell3d.faces = new_faces
```

Comment:
I would recommend not using for/else as it is not very readable.

Reply:
I understand. Then a better solution would be using a variable?